test/e2e_node/summary_test.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/onsi/gomega/types"
 )
 
-const restartCount = 3
+const restartCount = 1
```

Comment:
Ill add that.  I realize i did something similar in the [garbage collection test](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/garbage_collector_test.go#L163).  Ill refactor that and add it here quickly...